users/keys/mic92

```diff
@@ -1 +1,2 @@
-ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIKbBp2dH2X3dcU1zh+xW3ZsdYROKpJd3n13ssOP092qE joerg@turingmachine
+sk-ssh-ed25519@openssh.com AAAAGnNrLXNzaC1lZDI1NTE5QG9wZW5zc2guY29tAAAAIEVSsc5mlP8aWiUVwWWM3gKlB5LHVpmKSifnDyox/BnVAAAABHNzaDo= yubikey1
```

Comment:
https://github.com/nix-community/infra/blob/544c402c0a8de54820f7f2181a8d35610f6933f7/modules/darwin/community-builder/default.nix#L15

Security keys don't work ootb on darwin so I added support just for the darwin build box but we haven't confirmed that it works yet. If it does work we can move it to the common config so your key will work on both darwin hosts.

Reply:
Ok. I can test. Otherwise I can also switch to my tpm ssh key.

Reply:
Key actually doesn't work. Will switch to my tpm key instead.

Reply:
Thanks for testing. I've reverted the SecurityKeyProvider and reopened the original issue.

https://github.com/nix-community/infra/issues/1007, https://github.com/nix-community/infra/pull/1825